invenio_grobid/utils.py

```diff
@@ -26,7 +26,89 @@
 
```

Comment:
As in general, this should almost live into our overlay. What if we move this to a `contrib` sub dir and load it via the usual  `werkzeug` `import_string` pattern? (following a some config var in `invenio-grobid.config`?